scss/atoms/notifications/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+/*
+* ==========================================================================
+* File: _mixins.scss
+* For : Notifications
+* ==========================================================================
+*/
+
+@mixin notification($type) {
+	// General styles from
+	border-radius: map-get($notifications-element, border-radius);
+	margin: map-get($notifications-element, margin);
+	letter-spacing: map-get($notifications-element, letter-spacing);
+	font-weight: map-get($notifications-element, font-weight);
+	padding: map-get($notifications-element, padding);
+	font-size: map-get($notifications-element, font-size);
+	box-shadow: map-get($notifications-shadows, box-shadow-base);
+
+	// Type specific styles
+	background: map-deep-get($type, background, notification);
```

Comment:
Makes changing the variable value very difficult. the $type.background should ideally represent the color code itself, fetched from the palette